src/UI/Buyer/src/index.html

```diff
@@ -20,9 +20,39 @@
   </head>
 
   <body class="mat-typography">
-    <app-root></app-root>
+    <app-root id="app-root"></app-root>
   </body>
 </html>
+
+<script>
+  var prefix = '<PLACHOLDER>'
+  console.log('pathname: ' + document.location.pathname) // for debugging. Will remove later
+  console.log('window.location: ' + window.location.href)
+  if(document.location.pathname.split('/')[1] === prefix) {
+    const scriptItemsToReplace = ['runtime', 'polyfills', 'styles', 'main']
+    const linksToReplace = ['defaultbuyer']
+    var scripts = document.getElementsByTagName('script')
+    var links = document.getElementsByTagName('link')
+    for(item of scripts) {
+      var scriptSrc = item.getAttribute('src')
+      if(scriptSrc && !scriptSrc.includes(prefix) && scriptItemsToReplace.find(i => scriptSrc.includes(i))) {
+        var newScript = document.createElement('script')
+        newScript.setAttribute('src', prefix + '/' + scriptSrc)
+        document.body.appendChild(newScript)
+      }
+    }
+    for(item of links) {
+      var linkHref = item.getAttribute('href')
+      if(linkHref && !linkHref.includes(prefix) && linkItemsToReplace.find(i => linkHref.includes(i))) {
+        var newLink = document.createElement('link')
+        newLink.setAttribute('href', prefix + '/' + linkHref)
+        document.head.appendChild(newLink)
+      }
+    }
+  }
+
+</script>
```

Comment:
Not sure how its working currently but scripts should be within either head or body, in our case body. Can you move your script along with these other ones into the body?

Reply:
I can move the other script tags to inside the body. But the script I added only works correctly if it is outside the body tag. I think it's because all those other scripts get populated at the end of the body tag and so when my script runs they are not available yet. 
It works if it is kept outside the body though.

Reply:
Ok, yes please move the other ones then. 

I've been reading a bit and I think with this method we might still see a network request go out for the wrong scripts and then another one for the "fixed" scripts. If that happens we might need to refactor this so that initially there are no scripts populated and then we add them client-side.

Reply:
Sounds good. Just updated, and good point on the network requests for the wrong scripts. Think I might have an idea to take care of that.